examples/basic_bq/main.tf

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/******************************************
+   Provider configuration
+  *****************************************/
+provider "google" {
+  version = "~> 1.20.0"
+}
+
+module "bigquery" {
+  source            = "../.."
+  dataset_id        = "${var.dataset_id}"
```

Comment:
In an update, please make this example more useful by hardcoding most of these. Simply passing through variables doesn't actually make it any easier to understand the module.